openmc/mgxs/mgxs.py

```diff
@@ -4618,3 +4644,392 @@ def get_pandas_dataframe(self, groups='all', nuclides='all',
             df['std. dev.'] *= np.tile(densities, tile_factor)
 
         return df
+
+    def get_units(self, xs_type='macro'):
+        """Returns the units of Chi.
+
+        This method returns the units of Chi, which is "%" for both macro
+        and micro xs types.
+
+        Parameters
+        ----------
+        xs_type: {'macro', 'micro'}
+            Return the macro or micro cross section units.
+            Defaults to 'macro'.
+
+        Returns
+        -------
+        str
+            A string representing the units of Chi.
+
+        """
+
+        cv.check_value('xs_type', xs_type, ['macro', 'micro'])
+
+        # Chi has the same units (%) for both macro and micro
+        return '%'
+
+
+class ChiPrompt(Chi):
+    r"""The prompt fission spectrum.
+
+    This class can be used for both OpenMC input generation and tally data
+    post-processing to compute spatially-homogenized and energy-integrated
+    multi-group cross sections for multi-group neutronics calculations. At a
+    minimum, one needs to set the :attr:`ChiPrompt.energy_groups` and
+    :attr:`ChiPrompt.domain` properties. Tallies for the flux and appropriate
+    reaction rates over the specified domain are generated automatically via the
+    :attr:`ChiPrompt.tallies` property, which can then be appended to a
+    :class:`openmc.Tallies` instance.
+
+    For post-processing, the :meth:`MGXS.load_from_statepoint` will pull in the
+    necessary data to compute multi-group cross sections from a
+    :class:`openmc.StatePoint` instance. The derived multi-group cross section
+    can then be obtained from the :attr:`ChiPrompt.xs_tally` property.
+
+    For a spatial domain :math:`V` and energy group :math:`[E_g,E_{g-1}]`, the
+    fission spectrum is calculated as:
+
+    .. math::
+
+       \langle \nu\sigma_{f,\rightarrow g}^p \phi \rangle &= \int_{r \in V} dr
+       \int_{4\pi} d\Omega' \int_0^\infty dE' \int_{E_g}^{E_{g-1}} dE \; \chi(E)
+       \nu\sigma_f^p (r, E') \psi(r, E', \Omega')\\
```

Comment:
I think that more accurately reflects what the code actually does. I'll make the change.